lib/Heartbeat/LED.cpp

```diff
@@ -29,16 +29,18 @@
 
 #include "LED.h"
 
-LED::LED(int pin, bool activeHigh) {
+#define VAL(x) static_cast<uint8_t>((x) ? HIGH : LOW)
```

Comment:
`#define` is ugly.

Reply:
We could just make it an inline function. For me it doesn't matter as it does not leak from the compilation unit.

Reply:
inline function :+1:

Reply:
done